src/app/subnets/views/SubnetDetails/StaticDHCPLease/StaticDHCPLease.tsx

```diff
@@ -27,6 +31,7 @@ const StaticDHCPLease = () => {
           </Button>
         </MainToolbar.Controls>
       </MainToolbar>
+      <StaticDHCPTable staticDHCPLeases={getMockStaticDHCPLeases()} />
```

Comment:
This will need to be updated in order for tree-shaking to work correctly (make sure we only import mock data in development). As is, cooky-cutter is included in the production bundle increasing its size. Long term solution is to serve mock data using `msw`, but it's outside of scope of this. I'll followup with a PR to update it.